CHANGELOG.md

```diff
@@ -12,38 +12,47 @@ adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 ## [14.2.0] - 2024-03-06
 
 ### Added
-- Added new `beforeBeginEditing` hook [#10699](https://github.com/handsontable/handsontable/pull/10699)
-- Added an ability to prevent viewport scroll via `beforeViewportScrollVertically` and `beforeViewportScrollHorizontally` hooks. [#10724](https://github.com/handsontable/handsontable/pull/10724)
-- Undo will work for moving columns [#10746](https://github.com/handsontable/handsontable/pull/10746)
-- Added a `modifyFiltersMultiSelectValue` plugin hook and utilized it in the Filters' multi-selection component. [#10756](https://github.com/handsontable/handsontable/pull/10756)
-- Adds localization for Croatian language [#10762](https://github.com/handsontable/handsontable/pull/10762)
+
+- Added a new Handsontable hook, `beforeBeginEditing`, to conditionally control when to allow cell editing. [#10699](https://github.com/handsontable/handsontable/pull/10699)
+- Added the ability to prevent viewport scrolling by using two existing Handsontable hooks: `beforeViewportScrollVertically` and `beforeViewportScrollHorizontally`. [#10724](https://github.com/handsontable/handsontable/pull/10724)
+- Added undo-and-redo support for column moving. [#10746](https://github.com/handsontable/handsontable/pull/10746)
+- Added a new `Filters` plugin hook: `modifyFiltersMultiSelectValue`. Now, filtered values lists show formatted numbers instead of raw data. [#10756](https://github.com/handsontable/handsontable/pull/10756)
```

Comment:
List of filtered values should show the formatted numeric value not the source data.

```suggestion
- Added a new `Filters` plugin hook: `modifyFiltersMultiSelectValue`. Now, filtered values lists will show the formatted numeric value not the raw data. [#10756](https://github.com/handsontable/handsontable/pull/10756)
```